WordPress/Classes/ViewRelated/Reader/ReaderSiteStreamHeader.swift

```diff
@@ -89,12 +90,21 @@ fileprivate func > <T: Comparable>(lhs: T?, rhs: T?) -> Bool {
         }
     }
 
-    @objc func configureHeaderImage(_ siteBlavatar: String?) {
+    @objc func configureHeaderImage(_ siteBlavatar: String?, isWPForTeams: Bool = false) {
         let placeholder = UIImage.siteIconPlaceholder
+        let p2Placeholder = UIImage.gridicon(.p2)
```

Comment:
Since this is only used in one place, can it just be used directly? i.e. `avatarImageView.image = UIImage.gridicon(.p2)`